devtools/data/gen_serialized_results.py

```diff
@@ -208,31 +203,57 @@ def generate_ahfe_settings():
     settings.vacuum_equil_simulation_settings.production_length = 10 * unit.picosecond
     settings.vacuum_simulation_settings.equilibration_length = 10 * unit.picosecond
     settings.vacuum_simulation_settings.production_length = 1000 * unit.picosecond
-    settings.lambda_settings.lambda_elec = [0.0, 0.25, 0.5, 0.75, 1.0, 1.0,
-                                            1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0,
-                                            1.0]
-    settings.lambda_settings.lambda_vdw = [0.0, 0.0, 0.0, 0.0, 0.0, 0.12, 0.24,
-                                           0.36, 0.48, 0.6, 0.7, 0.77, 0.85,
-                                           1.0]
+    settings.lambda_settings.lambda_elec = [
```

Comment:
Please fmt off these - we let it happen in septop (where we need to revert things), this line-by-line list makes it really hard to quickly understand what the schedule looks like.